lib/cli/commands/build.js

```diff
@@ -100,6 +100,13 @@ build.builder = function(cli) {
 			describe: "Overrides the framework version defined by the project",
 			type: "string"
 		})
+		.option("experimental-css-variables", {
+			describe:
+				"If present, the CSS variables (css-variables.css, css-variables.source.less) " +
```

Comment:
Alternative proposal: `Generate CSS variables (css-variables.css, css-variables.source.less) and skeleton (library-skeleton(-RTL).css) for all themes`


I guess the "If present" part was taken over from the `clean-dest` flag. There however, it refers to the target directory and not the CLI flag.

Reply:
Thanks a lot @RandomByte. I've addressed your proposal in https://github.com/SAP/ui5-cli/pull/506.